### PR TITLE
[doc] System property required for JMX setup

### PR DESCRIPTION
**Pull request by honnix**
## Description
System property required for JMX setup.

Without it, JVM throws an exception complaining:

```
java.lang.RuntimeException: System property 'com.sun.management.jmxremote.rmi.port=null' does match configured RMI server port 9081
```

More specifically it is this line: https://github.com/airlift/airlift/blob/master/jmx/src/main/java/io/airlift/jmx/JmxAgent.java#L65

> Is this change a fix, improvement, new feature, refactoring, or other?
A doc improvement.

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)
No.

> How would you describe this change to a non-technical end user or system administrator?
NA

## Related issues, pull requests, and links
It was first reported by https://github.com/trinodb/trino/issues/7843

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by honnix** [comment]
_I'm working the CLA thing._

**Review by mosabua — APPROVED** [review]

File: docs/src/main/sphinx/admin/jmx.rst [hunk 1]
```diff
@@ -19,6 +19,13 @@ in the :ref:`config.properties file <config_properties>`:
   Specifies the port for the JMX RMI server. Trino exports many metrics,
   that are useful for monitoring via JMX.
 
+You also have to configure a system property in the
+:ref:`jvm.config file <jvm_config>`:
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/jmx.rst:23]:
```suggestion
:ref:`jvm.config <jvm_config>` with the RMI server port:
```

**Review by mosabua — CHANGES_REQUESTED** [review]
Minor edit .. but we also need a signed CLA to merge. Could you send that?

**Comment by honnix** [comment]
> Minor edit .. but we also need a signed CLA to merge. Could you send that?

Thank you for reminding me of this PR. I will try to get a CLA signed.

**Comment by mosabua** [comment]
Please squash the commits if possible and make sure you send a CLA.

**Comment by mosabua** [comment]
Can you rebase this into the new jmx.md file and submit a CLA?

**Comment by honnix** [comment]
Sure I can do that. Unfortunately the CLA is taking too much time to get. If you think this PR is useful, I don't mind at all you applying the change in another PR. @mosabua

**Comment by mosabua** [comment]
Linked PR is merged @honnix so this will go live with the 424 release

**Comment by honnix** [comment]
Thank you @mosabua !

**Pull request closed without merging**